terraform/fullnode/vultr/kubernetes.tf

```diff
@@ -0,0 +1,64 @@
+provider "kubernetes" {
+  config_path = local_file.kube_config.filename
+}
+
+resource "kubernetes_namespace" "aptos" {
+  metadata {
+    name = var.k8s_namespace
+  }
+}
+
+resource "kubernetes_storage_class" "ssd" {
+  metadata {
+    name = "ssd"
+  }
+  storage_provisioner = "block.csi.vultr.com"
+  volume_binding_mode = "WaitForFirstConsumer"
+  parameters = {
+    block_type = "high_perf"
+  }
+}
+
+provider "helm" {
+  kubernetes {
+    config_path = local_file.kube_config.filename
+  }
+}
+
+resource "helm_release" "fullnode" {
+  count            = var.num_fullnodes
+  name             = "${terraform.workspace}${count.index}"
+  chart            = "${path.module}/../../helm/fullnode"
+  max_history      = 100
+  wait             = false
+  namespace        = var.k8s_namespace
+  create_namespace = true
+
+  values = [
+    jsonencode({
+      chain = {
+        era = var.era
+      }
+      image = {
+        tag = var.image_tag
+      }
+      nodeSelector = {
+        "vke.vultr.com/node-pool" = "node"
```

Comment:
Is `node` a default name for node-pool on vultr? or should this value be the label you set above? which is `aptos-fullnode`

Reply:
Great catch. Will change it!

Reply:
Done